packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann"
+homepage: "https://github.com/andersfugmann/ppx-deriving-protocol"
+dev-repo: "git+https://github.com/andersfugmann/ppx-deriving-protocol"
+bug-reports: "https://github.com/andersfugmann/ppx-deriving-protocol/issues"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: [["jbuilder" "runtest" "-j" jobs]]
+version: "2.0.0"
```

Comment:
The `version` line can be removed from the `*.opam` files.  It is inferred based on the repository's directory structure.

Reply:
Ill remove them.